test/metabase/task/email_remove_legacy_pulse_test.clj

```diff
@@ -0,0 +1,36 @@
+(ns ^:mb/once metabase.task.email-remove-legacy-pulse-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.task.email-remove-legacy-pulse :as email-remove-legacy-pulse]
+   [metabase.test :as mt]
+   [toucan2.core :as t2]))
+
+(deftest send-emails-to-admins-if-instance-has-legacy-pulse
+  (mt/with-temp
+    [:model/User  admin  {:is_superuser true :first_name "Ngoc" :last_name "Khuat"}
+     :model/User  _user  {:is_superuser false}
```

Comment:
Why didn't you use `:crowberto` and `:rasta` here? We have test users so you don't need to create new ones.

Reply:
just for the sake of readability, If I use crowberto and rasta then it is not "clear" that it's intended to test this when there is at least one admin and 1 non-admin

Reply:
Fair enough. It should be sufficient to leave a comment explaining this but anyway, this approach is nice